kubeflow/pytorch-job/pytorch-operator.libsonnet

```diff
@@ -38,6 +38,18 @@
         subresources: {
           status: {},
         },
+        additionalPrinterColumns: [
+          {
+            JSONPath: ".status.conditions[-1:].type",
```

Comment:
Can conditions be empty?

Reply:
@richardsliu I verified that field will be empty in that case.